recipe/remove_unnecessary_warning.diff

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/draft/glpios03.c b/src/draft/glpios03.c
+index 21d6a00..eb34ad0 100644
+--- a/src/draft/glpios03.c
++++ b/src/draft/glpios03.c
+@@ -920,13 +920,10 @@ int ios_driver(glp_tree *T)
+ #if 0
+       ((glp_iocp *)T->parm)->msg_lev = GLP_MSG_DBG;
+ #endif
+-#if 1 /* 16/III-2016 */
++#if 1 /* 01/III-2018 */
+       if (((glp_iocp *)T->parm)->flip)
+-#if 0 /* 20/I-2018 */
+-         xprintf("WARNING: LONG-STEP DUAL SIMPLEX WILL BE USED\n");
+-#else
+-         xprintf("Long-step dual simplex will be used\n");
+-#endif
++         if (T->parm->msg_lev >= GLP_MSG_ALL)
++            xprintf("Long-step dual simplex will be used\n");
```

Comment:
Should we keep `Warning: `?

Reply:
Upstream does not retain the warning, I believe it should be removed.

Reply:
This is a patch from upstream that's already applied in their development repo, so I don't want to change it.